docs/api/menu.md

```diff
@@ -73,6 +73,8 @@ The `menu` object has the following instance methods:
 
 * `options` Object (optional)
   * `window` [BaseWindow](base-window.md) (optional) - Default is the focused window.
+  * `frame` [WebFrameMain](web-frame-main.md) (optional) - Provide the relevant frame
```

Comment:
Can there be a default value here in the same way that we provide the focused window for the `window` parameter above?

Reply:
I think this could be nice to add so all apps can take advantage of this feature by default. Especially in standard Electron use cases with one BrowserWindow containing a single WebContents. It might be worth adding a mention in the breaking changes doc too.

Reply:
I made the default `options.window.webContents.mainFrame` when possible, as that seems to be a sensible default.

Reply:
Oh I specifically didn't want to enable this by default because it's super tricky to guess the right frame. Especially in a window with WebContentsViews or iframes and such. This would also magically opt people into these features which they may not want 🤔

Reply:
Here's one more idea to make getting a default frame easier:
`webContents.getFocusedWebContents()?.focusedFrame`

I've added `focusedFrame` in https://github.com/electron/electron/pull/45667